orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -2887,9 +2887,14 @@ Return true if all elements are true and false otherwise.
           "Name of custom class.",
           AttributeProto::STRING)
       .Attr(
-          "call_convention",
-          "call_convention[i]==c means a non-tensor argument. call_convention[i]==d means a tensor.",
+          "input_convention",
+          "input_convention[i]==c means a non-tensor argument. input_convention[i]==d means a tensor.",
           AttributeProto::STRING)
+      .Attr(
+          "input_requires_grads",
+          "Flags to indicate whether the torch.autograd.apply's inputs (including both tensor"
+          " and non-tensor inputs) has gradient",
```

Comment:
```suggestion
          "Flags to indicate whether the torch.autograd.apply's inputs require gradients (including flags for both tensor"
          " and non-tensor inputs)",
```

Reply:
Thanks alot for the formatting, my clang-format did not work at some point, let me format them in batch. :)